ietf-optical-impairment-topology.yang

```diff
@@ -605,42 +635,316 @@ module ietf-optical-impairment-topology {
+           The max value correspond to worst case expected loss,including amplifier gain ripple or uncertainty";        
+      }
+      leaf roadm-minloss {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dB ;
+        description 
+          "The net loss from the ROADM input,to the output of the add/drop block.
+           If this path includes an amplifier, the amplifier gain reduces the net loss.  
+           This is before any additional drop path attenuation that may be required due to drop amplifier power contraints.
+           The min value correspond to best case expected loss,including amplifier gain ripple or uncertainty";        
+      }
+      leaf roadm-typloss {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dB ;
+        description 
+          "The net loss from the ROADM input,to the output of the add/drop block.
+           If this path includes an amplifier, the amplifier gain reduces the net loss.  
+           This is before any additional drop path attenuation that may be required due to drop amplifier power contraints.
+           The typ value correspond to typical case expected loss";        
+      }
+      leaf roadm-pmin {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dBm ;
+        description 
+          "If the drop path has additional loss that is added, for example, to hit target power levels into a 
+           drop path amplifier,or simply,to reduce the power of a “strong” carrier(due to ripple,for example), 
+           then the use of the ROADM input power levels and the above drop losses is not appropriate.
+           This parameter corresponds to the worst case per carrier power levels expected 
+           at the output of the drop block.To be noted that the net power levels from the drop block consider 
+           these power levels,and the power levels calculated with the drop loss attributes,if both are defined.
+           This comparison  captures the case if the ROADM input power levels are low enough such that extra
+           path loss is not required."; 
+      }
+      leaf roadm-pmax {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dBm ;
+        description 
+          "If the drop path has additional loss that is added, for example, to hit target power levels into a 
+           drop path amplifier,or simply,to reduce the power of a “strong” carrier(due to ripple,for example), 
+           then the use of the ROADM input power levels and the above drop losses is not appropriate.
+           This parameter corresponds to the best case per carrier power levels expected 
+           at the output of the drop block.To be noted that the net power levels from the drop block consider 
+           these power levels,and the power levels calculated with the drop loss attributes,if both are defined.

```

Comment:
same comment:
the sentence 'To be noted that ...' is a bit obscure when there is not the exact computation with it : maybe add that: a detail example of the comparison using these parameters is detailed in section xxx of the document xxx
blank after ','

Reply:
ok, as above